android/src/main/java/com/urbanairship/reactnative/ReactMessageView.java

```diff
@@ -106,7 +106,7 @@ void startLoading(final String messageId) {
             fetchMessageRequest = UAirship.shared().getInbox().fetchMessages(new RichPushInbox.FetchMessagesCallback() {
                 @Override
                 public void onFinished(boolean success) {
-                    RichPushMessage message = UAirship.shared().getInbox().getMessage(messageId);
+                    message = UAirship.shared().getInbox().getMessage(messageId);
```

Comment:
Do we need this mix of `this.message` and `message`? It's pretty confusing, and may have contributed to the problem. FWIW: I don't really care which one we use, but, if forced to choose, I'd say use `this.message`.

Reply:
`this.message` will not work in this context, that would refer to the callback. You would have to use `ReactMessageView.this.message`.